extensions/samples/electron-webpack/src/main.ts

```diff
@@ -0,0 +1,92 @@
+import { app, BrowserWindow } from "electron";
+import * as path from "path";
+import {
+  ElectronAuthenticator,
+  MsalElectronConfig,
+ } from '@microsoft/mgt-electron-provider/dist/Authenticator';
+ import { 
+  DataProtectionScope,
+  PersistenceCreator,
+  PersistenceCachePlugin,
+  IPersistence
+} from "@azure/msal-node-extensions";
+
+// You can use the helper functions provided through the Environment class to construct your cache path
+// The helper functions provide consistent implementations across Windows, Mac and Linux.
+const cachePath = "./cache.json";
```

Comment:
If the file in this location doesn't exist, will the cache plugin create it?

Reply:
Yes, if the file does not exist the cache file will be created.